intro/scipy/examples/solutions/plot_image_blur.py

```diff
@@ -41,14 +41,14 @@
 #####################################################################
 
 # Padded fourier transform, with the same shape as the image
-# We use :func:`scipy.signal.fftpack.fft2` to have a 2D FFT
-kernel_ft = sp.fftpack.fft2(kernel, shape=img.shape[:2], axes=(0, 1))
+# We use :func:`scipy.signal.fft.fft2` to have a 2D FFT
```

Comment:
```suggestion
# We use :func:`scipy.fft.fft2` to have a 2D FFT
```

Reply:
Ah didn't notice. Will commit before this is merged.